views_initiate_payment_gateway_transaction.py:10:0: C0301: Line too long (172/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:13:0: C0301: Line too long (101/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:20:0: C0301: Line too long (120/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:30:0: C0301: Line too long (108/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:42:0: C0301: Line too long (101/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:44:0: C0301: Line too long (103/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:51:0: C0301: Line too long (107/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:53:0: C0301: Line too long (102/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:60:0: C0301: Line too long (112/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:65:0: C0301: Line too long (115/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:74:0: C0301: Line too long (118/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:84:0: C0301: Line too long (109/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:87:0: C0301: Line too long (107/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:93:0: C0301: Line too long (120/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:95:0: C0301: Line too long (115/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:99:0: C0301: Line too long (104/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:103:0: C0301: Line too long (125/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:111:0: C0301: Line too long (118/100) (line-too-long)
views_initiate_payment_gateway_transaction.py:8:0: E0401: Unable to import 'common.apiauthentication.api.authenticateapi.views_authenticateapi' (import-error)
views_initiate_payment_gateway_transaction.py:9:0: E0401: Unable to import 'common.sessionmanagement.api.sessioncontrol.views_sessioncontrol' (import-error)
views_initiate_payment_gateway_transaction.py:10:0: E0401: Unable to import 'common.paymentgateways.api.initiate_payment_gateway_transaction.validations_initiate_payment_gateway_transaction' (import-error)
views_initiate_payment_gateway_transaction.py:11:0: E0401: Unable to import 'common.location.api.get_user_geos.views_get_user_geos' (import-error)
views_initiate_payment_gateway_transaction.py:12:0: E0401: Unable to import 'common.location.specificlib.views_lib' (import-error)
views_initiate_payment_gateway_transaction.py:13:0: E0401: Unable to import 'common.paymentgateways.specificlib.views_lib' (import-error)
views_initiate_payment_gateway_transaction.py:16:0: C0103: Constant name "logger" doesn't conform to UPPER_CASE naming style (invalid-name)
views_initiate_payment_gateway_transaction.py:49:4: R0914: Too many local variables (17/15) (too-many-locals)
views_initiate_payment_gateway_transaction.py:109:15: W0703: Catching too general exception Exception (broad-except)
views_initiate_payment_gateway_transaction.py:49:4: R0201: Method could be a function (no-self-use)


validations_initiate_payment_gateway_transaction.py:11:0: C0301: Line too long (103/100) (line-too-long)
validations_initiate_payment_gateway_transaction.py:12:0: C0301: Line too long (119/100) (line-too-long)
validations_initiate_payment_gateway_transaction.py:67:15: W0703: Catching too general exception Exception (broad-except)
validations_initiate_payment_gateway_transaction.py:9:4: R0911: Too many return statements (14/6) (too-many-return-statements)
validations_initiate_payment_gateway_transaction.py:9:4: R0912: Too many branches (13/12) (too-many-branches)